code/__DEFINES/layers.dm

```diff
@@ -142,19 +142,22 @@
 	#define LIGHTING_LAYER 1
 
 #define EFFECTS_ABOVE_LIGHTING_PLANE 5
+	#define EYEGLOW_LAYER 1
```

Comment:
Duplicate definition, there's the one below that is the same but with an underscore